background.js

```diff
@@ -75,10 +75,13 @@ function migrateSettings(version)
         }
     }
 
-    var current_version = chrome.app.getDetails().version;
+    // for Chrome
+    //var current_version = browser.app.getDetails().version;
```

Comment:
Do you need a browser check here?  Does the Chrome code need to run in Chrome?

Reply:
The version check seems to be for pulling up the change log when installing / updating the addon.

Reply:
I mean that the Chrome code path is commented out; I want to add Firefox support without breaking the Chrome support or needing to serve the two extensions from different codebases.  Can this commented-out code be executed when Chrome is running, and the new code executed when Firefix is running?

Reply:
I'm still investigating whether `chrome.app.getDetails().version` will work using a polyfill. Another approach may need to be used for cross-compatibility.

Reply:
I was picturing something dumb like `if (typeof chrome !== 'undefined') { /* chrome */ } else { /* firefox */ }`

Couldn't we mechanically support both browsers by just explicitly branching the logic based on whether `chrome` exists?  Then don't worry about trying to use literally the same code in both browsers.

Reply:
It appears workarounds like that are unnecessary as `webextension-polyfill` seems to work. Although, due to the deprecation of `sendResponse()` in the `onMessage` listener in `background.js` some Promise returns are necessary going forward.

I have a working cross-browser version of this PR in local testing. It also seems like the Options panel changes that work in Firefox also work fine in Chrome. Once I clean it up I'll push it for review.